TWLight/resources/models.py

```diff
@@ -328,6 +328,10 @@ class Meta:
         # Translators: In the administrator interface, this text is help text for a check box where staff can select whether users must first register at the organisation's website before finishing their application.
         help_text=_("Mark as true if this partner requires applicants to have "
                     "already signed up at the partner website."))
+    proxy_account_length = models.BooleanField(default=False,
```

Comment:
We maybe don't need this - I think ideally we would have this selector for all proxy partners.